Errors when building on i386

When building qjs for r2 on i386 arch, the following errors arise:

```
 #14 263.9 In file included from /usr/src/radare2-6.0.8/libr/..//subprojects/qjs/quickjs.c:46:0:
#14 263.9 /usr/src/radare2-6.0.8/libr/..//subprojects/qjs/quickjs.c: In function 'js_binary_arith_slow':
#14 263.9 /usr/src/radare2-6.0.8/libr/..//subprojects/qjs/cutils.h:643:5: error: a label can only be part of a statement and a declaration is not a statement
#14 263.9      unsigned short cw;                                                      \
#14 263.9      ^
#14 263.9 /usr/src/radare2-6.0.8/libr/..//subprojects/qjs/quickjs.c:13948:13: note: in expansion of macro 'JS_X87_FPCW_SAVE_AND_ADJUST'
#14 263.9              JS_X87_FPCW_SAVE_AND_ADJUST(fpcw);
#14 263.9              ^~~~~~~~~~~~~~~~~~~~~~~~~~~
#14 263.9 /usr/src/radare2-6.0.8/libr/..//subprojects/qjs/cutils.h:643:5: error: a label can only be part of a statement and a declaration is not a statement
#14 263.9      unsigned short cw;                                                      \
#14 263.9      ^
#14 263.9 /usr/src/radare2-6.0.8/libr/..//subprojects/qjs/quickjs.c:14020:9: note: in expansion of macro 'JS_X87_FPCW_SAVE_AND_ADJUST'
#14 263.9          JS_X87_FPCW_SAVE_AND_ADJUST(fpcw);
#14 263.9          ^~~~~~~~~~~~~~~~~~~~~~~~~~~
#14 268.3 make[4]: *** [/usr/src/radare2-6.0.8/libr/..//subprojects/qjs/quickjs.o] Error 1
#14 268.3 /usr/src/radare2-6.0.8/libr/../global.mk:43: recipe for target '/usr/src/radare2-6.0.8/libr/..//subprojects/qjs/quickjs.o' failed
```

Full log here: https://github.com/radareorg/radare2-snap/actions/runs/20697486056/job/59414905869

qjs version (commit): d405777f7eefa22c17c12970317ef3d6e7658f5a

--- a/quickjs.c
+++ b/quickjs.c
@@ -14103,9 +14103,11 @@ static no_inline __exception int js_binary_arith_slow(JSContext *ctx, JSValue *s
             }
             break;
         case OP_div:
-            JS_X87_FPCW_SAVE_AND_ADJUST(fpcw);
-            sp[-2] = js_number((double)v1 / (double)v2);
-            JS_X87_FPCW_RESTORE(fpcw);
+            {
+                JS_X87_FPCW_SAVE_AND_ADJUST(fpcw);
+                sp[-2] = js_number((double)v1 / (double)v2);
+                JS_X87_FPCW_RESTORE(fpcw);
+            }
             return 0;
         case OP_mod:
             if (v1 < 0 || v2 <= 0) {

